packages/astro/src/@types/astro.ts

```diff
@@ -2373,68 +2311,72 @@ export interface AstroUserConfig {
 		 * export const collections = { countries };
 		 * ```
 		 *
-		 * For more advanced loading logic, you can define an object loader. This allows incremental updates and conditional loading, and gives full access to the data store. See the API in [the RFC](https://github.com/withastro/roadmap/blob/content-layer/proposals/content-layer.md#loaders).
+		 * For more advanced loading logic, you can define an object loader. This allows incremental updates and conditional loading, and gives full access to the data store. See the API in [the draft Content Layer API RFC](https://github.com/withastro/roadmap/blob/content-layer/proposals/content-layer.md#loaders).
 		 *
-		 * ### Migrating an existing content collection to use the Content Layer API
+		 * #### Migrating a content collection to content layer
 		 *
-		 * You can convert an existing content collection to use the Content Layer API if it uses markdown, MDX or JSON, with these steps:
+		 * You can convert an existing content collection with Markdown, MDX, or JSON entries to use the Content Layer API.
```

Comment:
```suggestion
		 * You can convert an existing content collection with Markdown, MDX, Markdoc, or JSON entries to use the Content Layer API.
```
Just in case!